llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2173,8 +2171,7 @@ MemoryDepChecker::isDependent(const MemAccessInfo &A, unsigned AIdx,
   // We know that Dist is positive, but it may not be constant. Use the signed
   // minimum for computations below, as this ensures we compute the closest
   // possible dependence distance.
-  uint64_t MinDistanceNeeded =
-      TypeByteSize * *CommonStride * (MinNumIter - 1) + TypeByteSize;
+  uint64_t MinDistanceNeeded = *CommonStride * (MinNumIter - 1) + TypeByteSize;
```

Comment:
```suggestion
  uint64_t MinDistanceNeeded = *CommonStride * MinNumIter;
```

Reply:
This is an incorrect simplification. Kindly see comments preceding this line.

Reply:
I see now, thanks.